docs/config/setup/interfaces/mermaid.MermaidConfig.md

```diff
@@ -111,7 +111,31 @@ should not change unless content is changed.
 
 #### Defined in
 
-[packages/mermaid/src/config.type.ts:163](https://github.com/mermaid-js/mermaid/blob/master/packages/mermaid/src/config.type.ts#L163)
+[packages/mermaid/src/config.type.ts:188](https://github.com/mermaid-js/mermaid/blob/master/packages/mermaid/src/config.type.ts#L188)
+
+---
+
+### elk.mergeEdges
+
+• `Optional` **elk.mergeEdges**: `boolean`
+
+Elk specific option that allows egdes to share path where it convenient. It can make for pretty diagrams but can also make it harder to read the diagram.
+
+#### Defined in
+
+[packages/mermaid/src/config.type.ts:95](https://github.com/mermaid-js/mermaid/blob/master/packages/mermaid/src/config.type.ts#L95)
+
+---
+
+### elk.nodePlacement.strategy
```

Comment:
@knsv @ashishjain0512 Shouldn't these configs be proper objects?
Otherwise it will be very confusing, as we already support some configs in nested fashion. 

```yaml
config:
  elk:
    mergeEdges: true
    nodePlacement:
      strategy: SIMPLE
```